src/_includes/partials/components/filters.njk

```diff
@@ -0,0 +1,39 @@
+{% set filterableOptions = { projects: [], types: [] } %}
+{% for resource in collections['resources_' + lang] %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("projects", resource.data.projects) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("types", resource.data.type) %}
+{% endfor %}
+<div id="filters" class="filters flow">
+    <h3>{% __ 'filters' %}</h3>
+    <div class="flow">
+        <fieldset>
+            <legend class="visually-hidden">{% __ 'project' %}</legend>
+            {% disclosure 'project' %}
```

Comment:
We need to add icons but the `disclosure` shortcode doesn't support icons right now. @chosww You could re-implement this without using the shortcode, e.g. just putting in the `<inclusive-disclosure>` element in this loop.

Reply:
Hey Ned, I used different approach to render the icons on the disclosures; I used css to render the icons with mask-image properties, like how I did on the other parts of the page.